### ISSUE TITLE
Observer free code path for state

### ISSUE THREAD

**Issue by wied03** [zetachang/react.rb#136]
For those of us that use the GrandCentral/Redux approach and keep most state outside the component, we don't want to hit any `Observable` code for any incidental state calls. I'm assuming that any Observable code remains light to keep file sizes down.

**Comment by ajjahn** [comment]
@wied03 I agree that observable state shouldn't be required. This could be addressed in the state definition by passing an option. Not sure if it should be opt in or opt out, but I like declaring it for the purpose of having self documenting components.

**Comment by sollycatprint** [comment]
This issue was moved to reactrb/reactrb#136
